tsconfig.base.json

```diff
@@ -2,6 +2,8 @@
 	"compilerOptions": {
 		"composite": true,
 		"declaration": true,
+		"declarationMap": true,
+		"sourceMap": true,
```

Comment:
This is intentional too. I didn't want to include these things unnecessarily. I think we can approach them when we figure out tsdown, etc.

```suggestion
```